src/main/java/net/syscon/elite/api/resource/impl/BookingResourceImpl.java

```diff
@@ -215,11 +215,11 @@ public InmateDetail getOffenderBooking(final Long bookingId, final boolean basic
     @Override
     public InmateDetail getOffenderBookingByOffenderNo(final String offenderNo, final boolean fullInfo, final boolean extraInfo) {
 
-        final var bookingId = bookingService.getBookingIdByOffenderNo(offenderNo);
+        final var booking = bookingService.getBookingIdByOffenderNo(offenderNo);
```

Comment:
so is `getBookingIdByOffenderNo` returning booking now not an id? Should the method be renamed?

Reply:
So it returns a `OffenderBookingIdSeq` maybe rename as `getOffenderIdentifiers` or something